projects/demo/project.json

```diff
@@ -147,6 +147,13 @@
                 },
                 "development": {
                     "baseHref": "/"
+                },
+                "serve-fast": {
+                    "baseHref": "/",
+                    "sourceMap": false,
+                    "namedChunks": false,
+                    "statsJson": false,
+                    "budgets": []
```

Comment:
```json
"development": {
      "baseHref": "/"
      "sourceMap": false,
      "namedChunks": false,
      "statsJson": false,
      "budgets": []
  }
```


instead of

```json
"development": {
      "baseHref": "/"
  },
  "serve-fast": {
      "baseHref": "/",
      "sourceMap": false,
      "namedChunks": false,
      "statsJson": false,
      "budgets": []
  }
```

Reply:
New build  disabling source maps. In 85% of cases it's ok and you don't need it. But in 15%(as my experience said) you need to dive deeply  and debug over few components or directives(or so on). So in 85% of develop time you'll use new build. But in rest 15% of cases the old one.

Reply:
Maybe then default to fast build and add `start:sourcemap` for debugging?

Reply:
We can just

“start:sourcemap”: “ng serve —source-map true”